src/Gui/PreferencePacks/package.xml

```diff
@@ -9,7 +9,7 @@
 
   <content>
     <preferencepack>
-      <name>Classic</name>
+      <name>Freecad Classic</name>
```

Comment:
"FreeCAD" would be better than "Freecad"

Reply:
![image](https://github.com/user-attachments/assets/df17163b-3533-4af1-8317-e5ae2cb6f63f)
![image](https://github.com/user-attachments/assets/5e82c092-8007-4f60-b600-8f5c2c086ec6)